Assets/MixedRealityToolkit-SDK/Features/UX/Scripts/Tooltips/ToolTipBackground.cs

```diff
@@ -0,0 +1,56 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+using Microsoft.MixedReality.Toolkit.Core.Extensions;
+//using HoloToolkit.Unity;
```

Comment:
please remove... it is not a valid namespace in MRTK :)

Reply:
Cleaned up in my PR on his branch.